test/e2e/page-objects/pages/homepage.ts

```diff
@@ -84,11 +81,6 @@ class HomePage {
     await this.driver.clickElement(this.activityTab);
   }
 
-  async openAccountMenu(): Promise<void> {
-    console.log(`Opening account menu`);
-    await this.driver.clickElement(this.accountMenuButton);
-  }
-
```

Comment:
Intended change to remove the redundant `openAccountMenu` method, we should use the one in `header-navbar`.